### ISSUE TITLE
ERR! Windows_NT 6.1.7601

### ISSUE THREAD

**Issue by cmozbas** [npm/npm#12952]
I've a very ennoying problem since days and I pissed off because it blocked my visual studio 2015 solution, no more packages are loaded. So I just create a basic project then I saw the problem is coming from node install. Here What I do:
- Create a rubish folder and init a package file:

C:\Users\cmoz\Documents\Projects_rubish\02>npm init

package.json:
{
  "name": "02",
  "version": "1.0.0",
  "description": "sadf sd fasd",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [
    "test",
    "test1"
  ],
  "author": "coz",
  "license": "ISC"
}
- Then I try to install gulp
  npm install gulp –g 

ERROR !!!!!!!!!!!!!!!!!!!!!

C:\Users\cmoz\Documents\Projects_rubish\02>npm install gulp -g
npm WARN package.json 02@1.0.0 No repository field.
npm WARN package.json 02@1.0.0 No README data
npm ERR! addLocal Could not install C:\Users\cmoz\Documents\Projects_r
ubish\02-g
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\Users\cmoz\Documents\Programmes\nodejs\node.exe" "C:\Us
ers\cmoz\AppData\Roaming\npm\node_modules\npm\bin\npm-cli.js" "install"
"gulp" "-g"
npm ERR! node v4.4.5
npm ERR! npm  v2.12.1

npm ERR! The header content contains invalid characters
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     https://github.com/npm/npm/issues

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\cmoz\Documents\Projects\training_rubish\02\npm-debug.log

I searched on internet I saw some error but no solution. I tried many things, I list all what I did :
- Installed all version of node in x32 x64, last, old each version but still error
- Changed the installation directory but still error
- Installed the visual studio 2015 node tools but it require to install node so rubish
- Changing the path

If someone has a clue feel free to share. Thank you

**Comment by cmozbas** [comment]
More information about error. I think it's an internet access error. I'm using virtual box 4.20 then I updated it to 5.20, I tried with NAT, Bridge but in both case I got the same error. My internet work fine I don't see any problem why it block node command. I tried on my computer it worked perfectly but not in my virtualbox.

Here is the log file after error:

[npm-debug.txt](https://github.com/npm/npm/files/298836/npm-debug.txt)
